nav2_route/src/route_planner.cpp

```diff
@@ -192,4 +197,20 @@ bool RoutePlanner::isGoal(const NodePtr node)
   return node->nodeid == goal_id_;
 }
 
+bool RoutePlanner::isStart(const NodePtr node)
+{
+  return node->nodeid == start_id_;
+}
+
+nav2_route::EdgeType RoutePlanner::classifyEdge(const EdgePtr edge)
+{
+  if (isStart(edge->start)) {
+    return nav2_route::EdgeType::START;
+  } else if (isGoal(edge->end)) {
+    return nav2_route::EdgeType::END;
+  } else {
```

Comment:
Remove the `else` and just have `return EdgeType::NONE;` so that we remove a branch - its the default return if not start/goal. 

Also, you can remove the `nav2_route` namespace in the other conditions as well, this node is within that namespace already